src/applications/vaos/appointment-list/components/AppointmentsPageV2/RequestAppointmentLayout.jsx

```diff
@@ -68,9 +71,6 @@ export default function RequestAppointmentLayout({ appointment }) {
                     {`${modality}`}
```

Comment:
modality contains the returned value from the function `selectModalityText(appointment)` Currently it returns  an in-person upcoming appointment with "At ${facilityName}" or "At VA facility". However, based on the pending AC, pending requests for an in-person should return "In Person" not "At VA facility".  Maybe create an overload for selectModalityText(appointment, isRequest) such that if isRequest and isInPerson then return "In Person"

Reply:
Done.